docker/local_development/zabbix_monitoring_centos_local_dev.yaml

```diff
@@ -0,0 +1,374 @@
+apiVersion: v1
```

Comment:
Consider using the non-local development zabbix monitoring template using passed in vars.